hack/preview.sh

```diff
@@ -112,6 +121,12 @@ update_patch_file () {
 update_patch_file "${ROOT}/argo-cd-apps/k-components/inject-infra-deployments-repo-details/application-patch.yaml"
 update_patch_file "${ROOT}/argo-cd-apps/k-components/inject-infra-deployments-repo-details/application-set-patch.yaml"
 
+# Add monitoringstack to $ROOT/components/monitoring/prometheus/development/kustomization.yaml to deploy Observability operator
```

Comment:
Not sure we need this comment.

Reply:
I removed the comment and updated the prompt